app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.java

```diff
@@ -284,8 +286,20 @@ public static void settingsEvent(String key, String value) {
                 .queue();
     }
 
-    public static void settingsClickEvent(String key) {
+    public static void settingsClickEvent(Context context, String key) {
         TelemetryEvent.create(Category.ACTION, Method.CLICK, Object.SETTING, key).queue();
+
+        if (context == null) {
+            return;
+        }
+        String pref_key_storage_save_downloads_to = context.getString(R.string.pref_key_storage_save_downloads_to);
```

Comment:
Just curious: If we're only interested in these events, why not also apply this key check for Telemetry?

Reply:
In the document, only a subset of events is sent to Firebase. So I exclude others here.